src/components/Editor/Preview/index.js

```diff
@@ -106,8 +106,6 @@ class Preview extends PureComponent {
   }
 }
 
-Preview.displayName = "Preview";
```

Comment:
can we try to remove all the displayName propeties in this PR?

Reply:
yes going to do all of them in this one